app/views/users/show.html.haml

```diff
@@ -0,0 +1,36 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(@user.email.downcase)}?s=40")
+          %h2 #{@user.full_name} video collections (#{@user.queue_items.count})
+          = link_to("Follow", relationships_path(leader_id: @user.id), class: "btn btn-default", method: :post) if current_user.can_follow?(@user)
+        %table.table
+          %thead
+            %tr
+              %th(width="30%") Video Title
+              %th(width="15%") Genre
+          %tbody
+            - @user.queue_items.each do |queue_item| 
+              %tr
+                %td
+                  = link_to(queue_item.video_title, video_path(queue_item.video))
```

Comment:
Thanks!!
 I prefer the video_path(queue_item.video) style because it is clearer to me.
But I will remember that there is a conciser version. 
Thank again!!